src/Aspire.Dashboard/Components/Controls/LogViewer.razor.css

```diff
@@ -34,21 +34,19 @@
     background: var(--console-background-color);
     color: var(--console-font-color);
     font-family: 'Cascadia Mono', Consolas, monospace;
-    font-size: 12px;
+    font-size: 13px;
```

Comment:
I'm fine with changing it to 13px, just noting that 12px is what AzDO and GH Actions both use so that's why it was picked.

Reply:
It felt small to me compared to other text. And there was feedback in the line number color issue that text was small.

I tried 14px and it was too big. 13px felt good. IMO stick with it, the the app out, and see if we want to change its size again.